spec/acceptance/app_spec.rb

```diff
@@ -220,7 +220,7 @@ def app
 
   context "Stats page" do
     before do
-      stub_request(:get, "https://api.github.com/search/issues?q=is:pr%20user:alphagov%20author:app/dependabot")
+      stub_request(:get, "https://api.github.com/search/issues?q=is:pr%20user:alphagov%20state:open%20author:app/dependabot%20archived:false")
```

Comment:
Can you explain (in the commit message) why these changes are necessary? Does the Octokit SDK now append these extra parameters to requests?